CONTRIBUTING.md

```diff
@@ -64,6 +65,32 @@ issue yourself. This gives the team visibility of the work you plan to do.
 The other advantage to raising the issue at the outset is that our process requires an
 issue to be associated with every PR (see [patch format](#patch-format)).
 
+### Before submitting a PR
+
+Kata Containers utilizes a number of [CI systems](https://en.wikipedia.org/wiki/Continuous_integration)
+to automatically check every PR. As well as performing execution tests locally on
+your PR before submission, you are also strongly encouraged to run the suite of
+[static checks](https://github.com/kata-containers/tests/blob/master/.ci/static-checks.sh)
+the CI will execute. The easiest way to execute these in most of the repositories is
+to invoke the `Makefile` `check` rule, whilst enabling developer mode:
+
+```sh
+$ export KATA_DEV_MODE=true
+$ make check
+```
+
+Execution of this should complete with no errors. If errors are reported, fix them
```

Comment:
Lines 82-83 suggested rewrite: 
This should execute without errors. If errors are reported, fix them before you submit your PR.